src/Sylius/Bundle/ThemeBundle/spec/Asset/Package/UrlPackageSpec.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ThemeBundle\Asset\Package;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\ThemeBundle\Asset\PathResolverInterface;
+use Sylius\Bundle\ThemeBundle\Context\ThemeContextInterface;
+use Sylius\Bundle\ThemeBundle\Model\ThemeInterface;
+use Symfony\Component\Asset\PackageInterface;
+use Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface;
+
+/**
+ * @author Kamil Kokot <kamil@kokot.me>
+ * @author Stefan Doorn <stefan@efectos.nl>
+ */
+final class UrlPackageSpec extends ObjectBehavior
+{
+    function let(
+        VersionStrategyInterface $versionStrategy,
+        ThemeContextInterface $themeContext,
+        PathResolverInterface $urlResolver
+    ): void {
+        $this->beConstructedWith(['https://cdn-url.com/'], $versionStrategy, $themeContext, $urlResolver);
+    }
+
+    function it_implements_package_interface(): void
```

Comment:
We might also add `it_extends_symfony_url_package` spec.

Reply:
Added.